src/main/java/core/basesyntax/data/Formater.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.data;
+
+import core.basesyntax.transaction.FruitTransaction;
+
+public interface Formater {
```

Comment:
The interface name 'Formater' should be corrected to 'Formatter' to reflect proper English spelling.